Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3-buster as base
+WORKDIR /usr/src/app
+ENV VIRTUAL_ENV=/usr/src/app/venv
+RUN python -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+RUN pip install gunicorn
```

Comment:
Instead of these lines, you could just add gunicorn to your Poetry dependencies (`poetry add gunicorn`) so that it gets installed by the `poetry install` step.

You may then need `"poetry", "run", "gunicorn" ...` in your CMD.